### ISSUE TITLE
rewinding train time

### ISSUE THREAD

**Issue by L-JunJie** [ChengHan111/E2VPT#6]
Why does the rewinding process take so long？

**Comment by ChengHan111** [comment]
Thanks for your interest in our work. The rewinding takes place at the final step of E2VPT, which should definitely take a while to search over the candidate space (with different hyper-parameter settings).

**Comment by L-JunJie** [comment]
Hi author, can you provide the hyperparameters for each dataset, including learning rate, etc.

**Comment by ChengHan111** [comment]
I think I have provided some of them as examples in the command.txt. However, I am afraid that I don't have time to calibrate every single one of them at this moment. You might follow the VPT's setting and shrink the length of visual prompt to get the exactly same performance as reported in the paper.
